pkg/apis/mdc/v1alpha1/zz_generated.openapi.go

```diff
@@ -67,7 +67,23 @@ func schema_pkg_apis_mdc_v1alpha1_MobileDeveloperConsoleSpec(ref common.Referenc
 		Schema: spec.Schema{
```

Comment:
If you think these generated files are not useful in reviews, you can copy the `.gitattributes` file from the root of the unifiedpush-operator repo, and then GitHub will collapse (not show) them. It's nbd either way though :smiley_cat:

Reply:
Thanks! Done